includes/datastream.inc

```diff
@@ -138,6 +143,7 @@ function islandora_rest_datastream_post_response(array $parameters) {
   $datastream->label = isset($request['label']) ? $request['label'] : $file['name'];
   $datastream->versionable = isset($request['versionable']) ? (bool) $request['versionable'] : TRUE;
   $datastream->mimetype = isset($request['mimeType']) ? $request['mimeType'] : $mime_detect->getMimetype($file['name']);
+  $datastream->checksumType = isset($request['checksumType']) ? $request['checksumType'] : 'DISABLED';
```

Comment:
Seems like rather than setting it to `'DISABLED'` outright, we could probably `variable_get()` `islandora_checksum_checksum_type` for consistency's sake? I believe the default for that variable is `'DISABLED'` anyway.